rpc_jobs/irr_role_test.yml

```diff
@@ -15,8 +15,37 @@
           IMAGE: "Ubuntu 16.04 LTS (Xenial Xerus) (PVHVM)"
       - trusty:
           IMAGE: "Ubuntu 14.04 LTS (Trusty Tahr) (PVHVM)"
+    # NOTE(cloudnull): Context are used to set the OpenStack series used in the gate.
```

Comment:
"context" is the correct name as it defines the context in which the gate job will test the playbooks.